terasoluna-gfw-common-libraries/terasoluna-gfw-web/pom.xml

```diff
@@ -86,8 +86,8 @@
     <!-- == Begin Tomcat == -->
     <!-- Servlet API 3.0 -->
```

Comment:
- [ ] `Servlet API 4.0` is correct.

Reply:
fix.
7b360633dc20bfd69a68473330daabec38e4b449